tests/lora/tpu/test_pallas_kernels.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: Apache-2.0
+import jax
+import jax.numpy as jnp
+import numpy as np
+import pytest
```

Comment:
mostly curious, but why the jax implementation here?

Reply:
Ah I was using that earlier on before integrating the kernel with pytorch, switching over now